ci/axis/miniforge-cuda-arm64.yml

```diff
@@ -20,7 +20,6 @@ ARCH_TYPE:
 CUDA_VER:
   - 11.5.1
   - 11.4.1
-  - 11.3.1
```

Comment:
there are some more `11.3.1` references towards the bottom of this file that can be removed.